bin/mtx_to_h5ad.py

```diff
@@ -28,6 +31,40 @@ def mtx_to_adata(
     return adata
 
 
+def write_counts(
+    adata: AnnData,
+    txp2gene: str,
+    star_index: str,
+    out: str,
+    verbose: bool = False,
+):
+
+    if verbose:
+        print("Reading in {}".format(txp2gene))
+
+    features = pd.DataFrame()
+    features["id"] = adata.var.index
+
+    # if txp2gene file is available enrich features file with gene names
+    if txp2gene:
+        t2g = pd.read_table(txp2gene, header=None)
+        id2name = {e[1]: e[2] for _, e in t2g.iterrows()}
+        features["name"] = adata.var.index.map(id2name)
+
+    # if star_index file is available enrich features file with gene names
+    if star_index:
+        t2g = pd.read_table(f"{star_index}/geneInfo.tab", header=None, skiprows=1)
+        id2name = {e[0]: e[1] for _, e in t2g.iterrows()}
+        features["name"] = adata.var.index.map(id2name)
```

Comment:
I would suggest to first assign the gene_id to `adata`. It would be great to have the gene symbol also in the `h5ad` file. 

Nitpick: using a dictionary and `map` is fine, but there's a more compact and efficient solution: 

```python
if txp2gene: 
   t2g = pd.read_table(txp2gene, header=None, names=["FIRST_COL", "transcript_id", "gene_symbol"])
elif star_index:
   t2g = pd.read_table(star_index, header=None, names=["transcript_id", "gene_symbol"])

# when assigning a pandas column to a dataframe, pandas already takes care of matching the index. 
# therefore, setting the index is enough. 
t2g.set_index("transcript_id", inplace=True)
adata.var["gene_symbol"] = t2g["gene_symbol"]

# Now proceed exporting `adata.var` as "features"
```